### DISCUSSION TITLE
Why is it that even after training for over 70 epochs (766k steps), the generated audio in TensorBoard still isn't clear?

### DISCUSSION THREAD

**Discussion by EobardThawne721** [p0p4k/vits2_pytorch#87] [Q&A] [answered]
I trained for 766k steps using the same configuration but without SDP, yet the audio generated in TensorBoard is still unclear, with words being indistinct and muddled

**Comment by p0p4k** [comment]
I cannot debug for free with so little information  =)

**Reply by EobardThawne721** [reply, marked as answer]
anyway,I am still very grateful for the code you provided, which has been of great help to me!

**Reply by p0p4k** [reply]
Thanks, next time provide some more info so I or someone else can help you better. Good luck! =)
